webapp-django/crashstats/signature/static/signature/js/signature_tab_summary.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Tab for displaying signature summary.
+ * Does not have any panels.
+ * Does not have any controls.
+ *
+ * @extends {SignatureReport.Tab}
+ * @inheritdoc
+ */
+SignatureReport.SummaryTab = function(tabName) {
+
+    var config  = {
+        'panels': false,
+        'dataDisplayType': 'table',
+        'pagination': false
+    };
+
+    SignatureReport.Tab.call(this, tabName, config);
+
+};
+
+SignatureReport.SummaryTab.prototype = SignatureReport.inherit(SignatureReport.Tab.prototype);
```

Comment:
Some day we'll write ES6 code and this will look a lot better. This is the result of Thalia's work, and it's much better than before (all tabs used to be in one big file). I think it can certainly be made better, but that doesn't look like a priority for now.